07_Bootstrap/Praktikum/index.html

```diff
@@ -0,0 +1,184 @@
+            <div class="container-fluid">
+                    <a class="navbar-brand" href="#"><span>e</span>Startup</a>
+                    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+                      <span class="navbar-toggler-icon"></span>
+                    </button>
+              <div class="collapse navbar-collapse justify-content-end" id="navbarNavDropdown">
+                <ul class="navbar-nav">
+                  <li class="nav-item">
+                    <a class="nav-link active" aria-current="page" href="#">Home</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link" href="#">About</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link" href="#">Features</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link" href="#">Screenshots</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link" href="#">Team</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link" href="#">Pricing</a>
+                  </li>
+                  <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                      Drop Down
+                    </a>
+                    <ul class="dropdown-menu">
+                      <li><a class="dropdown-item" href="#">Action</a></li>
+                      <li><a class="dropdown-item" href="#">Another action</a></li>
+                      <li><a class="dropdown-item" href="#">Something else here</a></li>
+                    </ul>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link" href="#">Contact</a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </nav>    
+    </section>
+    <section class="content-1 text-center">
+        <p class="text-simple fs-1 fw-lighter ">Simple Systeme Fordiscount</p>
+        <p class="text-integer mt-3">Integer Cursus Bibendum Augue Ac Cursus</p>
+        <div class="content-column">
+            <div class="container text-center">
+                <div class="row">
+                  <div class="col">
+                    <p><img src="assets/images/cloud.svg" alt="cloud.png" style="width: auto; height:50px" > </p>
+                        <p class="fw-semibold">INTRODUCING WHATSAPP </p>
+                        <p class="fw-lighter">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Quam placeat
+                        ullam</p>
+                        <p>Read More</p>
+                  </div>
```

Comment:
kalo pengen pake symantic html untuk box image bisa pake figure > img > figcaption

reference:
[HTML `<figcaption>` Tag - w3school](https://www.w3schools.com/tags/tag_figure.asp)
[`<figcaption>`: The Figure Caption element - mdn web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)